### DISCUSSION TITLE
Security Onion in Hyper V

### DISCUSSION THREAD

**Discussion by mondster2602** [Security-Onion-Solutions/securityonion#3921] [Unsupported Versions]
Hi!

I would like to ask if someone already tried installing security onion in hyper-v?  What are the issues encountered during and after the installation, if there is any.

Thank you!

**Comment by Mastadamus** [comment]
I have my security onion installed using hyper v.
If doing a nic as promiscuous you have to create a virtual switch with promiscuous mode in powershell and attach it to the vm.  I used the sec onion iso.

**Reply by rrbinda** [reply]
Hi, how many virtual switches do you have in your set up? 
I have 4 vSwitches: WAN (external), LAN, VICTIM and SPAN as private. I have a pfsense VM attached to these switches and I created a bridge having the VICTIM as a member and the SPAN as the span port. Traffic from victim is comming to the span interface. I connected the SecOnion VM to the LAN and SPAN switches, where the management is using the LAN and the monitoring the SPAN. However SecOnion is not getting any data in SPAN interface. I also configured the port mirroring (pfsense/SPAN as source and SecOnion/span as destination), but data is still not comming.

**Reply by NocteDefensor** [reply]
I have 4 switches on my security onion. 1 is the admin/management switch, 1 is a private virtual switch mirroring traffic from each VM in my DMZ. The other is a private virtual switch mirroring traffic from each VM in my virtual internal "secure" network, and the other is a mirror for my physical ubiquiti switch span port.
